siliconcompiler/sphinx_ext/dynamicgen.py

```diff
@@ -218,33 +218,14 @@ def run(self):
     def get_modules(self):
         '''Gets dynamic modules under `self.PATH`.
 
-        This function explicitly searches builtins as well as SCPATH
-        directories. Although the directory for builtin tools gets added to
-        SCPATH after a chip object has been initialized, we can't rely on this
-        since we can't be sure that's happened yet. Therefore, we have to check
-        each one explicitly.
-
-        However, this could result in duplicate modules being detected once the
-        SCPATH does get updated. Therefore, we check to ensure that SCPATH
-        directories are not equal to the builtins directory before searching it.
-
-        TODO: we want better duplicate resolution (in case the user explicitly
-        declares a duplicate tool), where SCPATH takes priority.
+        This function explicitly searches builtins.
```

Comment:
some functionality like this might be useful to preserve

Reply:
we use this for out internal documentation generation

Reply:
Where should we search instead of `SCPATH` then though?
All dependencies?

Reply:
We'll figure it out